### DISCUSSION TITLE
Feature request: Localization settings (time and date, numbers, currency), networks

### DISCUSSION THREAD

**Discussion by juanriccio** [farag2/Sophia-Script-for-Windows#284] [Ideas]
Hello, I've been using Sophia Script on three machines: a new one that served as the "template" and two following computers I reformatted and brought to Windows 10. Among the settings I had to replicate by hand were the following:

- Localization: Time and date format, number format (decimal point vs. comma etc.)
- Networks: private/public, metered/unmetered. I like to keep my networks set up as "metered" so Windows doesn't go wild downloading and installing updates without my consent.

There are more settings of course, but these are the ones I think could be best managed automatically by Sophia Script. Have you ever considered adding them?

**Comment by farag2** [comment]
I was asked about this request a few times before, but I have no plans for inplementing these features into the script, because I don't think it's necessary for this project. This script is not a mix of features as many think, this is for personazitioning only. :)

**Reply by juanriccio** [reply]
I consider localization etc. as personalization settings indeed, but of course you are the only judge as to what's appropriate to add or not. Thank you for a highly effective utility anyway! :-)
